modules/cnv_install_1_4.adoc

```diff
@@ -0,0 +1,64 @@
+[[install_cnv]]
+=== Installing {ProductName} to {product-title}
+
+The `kubevirt-ansible` RPM contains the latest automation for deploying 
+{ProductName}. This procedure installs all {ProductName} components to your 
+{product-title} cluster. 
+
+This procedure installs the following components:
+
+* {ProductName} core components (KubeVirt)
+* Containerized data importer (CDI) controller
+* Multus, Open vSwitch (OVS), and SR-IOV container network interface plug-ins
+* Updated {product-title} web console
+
+.Prerequisites
+* A running {product-title} 3.11 cluster
+* User with `cluster-admin` privileges
+
+.Procedure
+
+. Log into the {product-title} cluster as an admin user:
++
+----
+$ oc login -u system:admin
+----
+
+. Install the `kubevirt-ansible` RPM and its dependencies:
++
+----
+$ yum install kubevirt-ansible
+----
+
+. If your cluster has unique requirements, customize the 
+`/usr/share/ansible/kubevirt-ansible/vars/cnv.yml` file. Otherwise, skip this 
```

Comment:
Duplicating code is usually a bad practice. Why would we want to promote it?

The only thing that we can possibly support are few edits to cnv.yml. Modifying roles or playbook should not be documented, nor keeping a possibility-stale copy of them.

Reply:
> Duplicating code is usually a bad practice.

Agree 100%, bad practice no doubts.

> The only thing that we can possibly support are few edits to cnv.yml. Modifying roles or playbook should not be documented, nor keeping a possibility-stale copy of them.

+1 :+1:

Reply:
I tend to agree here, if we end up copying something, that's usually a bad practice (e.g. copy of all of kubevirt-ansible).
can you:
* explain the motivation to copy the files? is it just how ansible searches for variables? 
* explain how often does a user end up changing the defaults? 

my motivation is to reduce the permutation of testing / verification to a minimum, if we can be opinionated instead of expose all of the playbook internals, the better. after all, this is our "api" to the user.

Reply:
@dankenigsberg All the playbooks and roles from `kubevirt.yml`.  This will look for the `kubevirt` role in `./roles` and in `/etc/ansible/roles`.  If you copy only cnv.yml, the roles will be in neither.  Perhaps the best solution here is to install all our roles and playbooks in the default ansible location.

```yaml
....
# Deploy kubevirt
- name: Deploy kubevirt role
  hosts: localhost
  connection: local
  gather_facts: False
  # unset http_proxy. required for running in the CI
  environment:
    http_proxy: ""
  roles:
    - role: kubevirt

# Deploy kubevirt web UI
- import_playbook: kubevirt_web_ui.yml
when: platform == "openshift"
...
```

Reply:
1. I don't want to copy `cnv.yml`. I'd like users to either specify their modification in the command line or write their own variable file.
2. I don't understand why using a `cnv.yml` from my home dir can possibly cause playbooks/kubevirt.yml to change the paths it uses. Are you sure you're solving a real problem? can you share an error log?

Reply:
forget it.  My config file was changing the default path. Non-issue.